src/App.tsx

```diff
@@ -22,6 +23,7 @@ export default function App() {
   return (
     <BrowserRouter>
       <Routes>
+        <Route path="/" element={<Login />} />
```

Comment:
로그인 페이지는 개발 단계에서만 필요하기 때문에 조건을 넣는 게 좋을 거 같습니다.
공통 컴포넌트로 래퍼 함수를 만들어서 감싸는 것도 좋은 방법일 거 같습니다. (간단한게 유틸 함수 만들어도 좋을 거 같습니다.

vite에서는 환경변수 내장 상수로 `import.meta.env.MODE` 를 지원하고 있어서 이걸 활용해서 개발 모드인지 확인할 수 있습니다. ([참고문서](https://ko.vite.dev/guide/env-and-mode.html#modes))
```suggestion
        <Route path="/" element={<DevOnly><Login /><DevOnly/>} />
        
// 주의 - 타입 별도 추가 필요
export default function DevOnly({children, fallback = null}) {
  if(import.meta.env.MODE !== 'development') {
    return fallback
  }
  return children
} 
```

Reply:
vite에 이런 기능들도 있었네요!

문서 보면서 `import.meta.env.DEV` 개발환경인지에 따라 boolean 값을 반환해주는 기능이 있어서 해당 기능 활용해서 wrapper 추가해봤습니다!
[cc3b29e](https://github.com/BCSDLab/KOIN_ORDER_WEBVIEW/pull/138/commits/cc3b29e2bc3060646fe4c534faebe0aafd39999c)